app/src/main/java/com/github/se/icebreakrr/model/tags/TagsViewModel.kt

```diff
@@ -0,0 +1,3 @@
+package com.github.se.icebreakrr.model.tags
+
+data class TagsViewModel(val repository: TagsRepository) {}
```

Comment:
Question: why is it empty and unused in the main screen?

Reply:
As I said in my PR, it will be in my next PR, if I do the viewModel in this one there will be too much change for a PR.

Reply:
Oh you're right. As repository is not used really in this PR, you could just remove it from mainscreen and replace it with a comment mentioning your intention of adding the VM later